CAN-1419,CAN-1439,CAN-1440

vikasranatalentelgia opened the pull request:
Warning is missing while delegating support
Bug: Drag and drop is not working
Zip code when enter and update, the value is not saving

vikasranatalentelgia committed:
CAN-1419,CAN-1439,CAN-1440

Warning is missing while delegating support
Bug: Drag and drop is not working
Zip code when enter and update, the value is not saving

akuks commented on src/components/ComponentPages/DirectSupportedCamps/DirectSupportedCampsUI/index.tsx:
```diff
@@ -128,12 +128,16 @@ export default function DirectSupportedCampsUI({
-                                {tag.camp_name}
-                              </a>
-                            </Link>
+                            <a
```
We need to use Link here

SidhantUppalSyantaq replied:
@akuks Link is causing an issue in the drag-and-drop functionality which is why I have removed the link here

vikasranatalentelgia replied:
@akuks Link is causing an issue in drag and drop functionality which is why I have removed the link here

vikasranatalentelgia committed:
Merge branch 'development' into Fixes

vikasranatalentelgia committed:
CAN-1441,CAN-1434

Pagination is missing
User A is not able to delegate support to User B in below case
